MdeModulePkg/Universal/SetupBrowserDxe/Presentation.c

```diff
@@ -252,7 +252,13 @@ CreateRefreshEventForStatement (
   ASSERT_EFI_ERROR (Status);

```

Comment:
In the spirit of what's being done, these assert only on errors in the files you've changed should probably be converted as well since it's a quick change.

Reply:
I'm super burned on CodeQL fixes after doing several hundred over the last month and would like to wrap up this batch 😅